packages/oruga/src/components/button/tests/Button.browser.test.ts

```diff
@@ -0,0 +1,20 @@
+import { describe, expect, test } from "vitest";
+
+import { render } from "vitest-browser-vue";
+
+import OButton from "../Button.vue";
+
+describe("<Button>", () => {
+    test("render button text", async () => {
+        const screen = render(OButton, {
+            slots: {
+                default: () => "Hello World!",
+            },
+        });
+        const button = screen.getByRole("button");
+        await expect.element(button).toHaveTextContent("Hello World!");
```

Comment:
why do you need to await the expect here?

Reply:
`expect.element` is syntactic sugar for [`expect.poll`](https://vitest.dev/api/expect#poll), so it repeatedly (and asynchronously) polls the DOM until the condition is met or it times out. See [the API docs](https://vitest.dev/guide/browser/assertion-api.html) for usage notes.